src/components/KToaster/ToastManager.ts

```diff
@@ -43,18 +43,18 @@ export default class ToastManager {
     createApp(Toast).mount(`#${this.id}`)
   }
 
-  setTimer(key: any, timeout: number) {
+  setTimer(key: any, timeout: number): number {
```

Comment:
question: why is the return type `number` here?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#return_value